core/src/main/java/org/mobilitydata/gtfsvalidator/annotation/GtfsValidationNotice.java

```diff
@@ -0,0 +1,80 @@
+package org.mobilitydata.gtfsvalidator.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.mobilitydata.gtfsvalidator.notice.SeverityLevel;
+import org.mobilitydata.gtfsvalidator.table.GtfsEntity;
+
+/**
+ * An annotation used to identify, configure, and document a validation notice. All notices should
+ * be annotated with this type.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface GtfsValidationNotice {
```

Comment:
question: Do you think there is value in adding the validator class as a field in the annotation? Add it might help us to identify programmatically easier the relation between validators and the notices.

Reply:
Perhaps?  I know some generic Notices are used by more than one validator.  That said, most Notices are defined as static inner classes of the validator itself.  In the spirit of preferring convention over configuration, we can probably extract that relationship automatically to determine the linkage between validator and notice.

Reply:
The `convention over configuration` is a great point! Thanks!